js/schematic.js

```diff
@@ -101,6 +99,19 @@ function corpus(contribs, contribs2, contribs3, disclosure1, disclosure2, disclo
         .tooltip(tip)
       () // Call grid()
     ;
+    var years = d3.extent(data, function(d){ return +d.Year; });
+    // Populate Year Selector
+    d3.select("#chooser-year")
```

Comment:
This might be a good place to add some logic to update the dropdown when the year was changed via a different interaction (clicking on the year in the grid, or clicking on a cell in the grid).